parts/k8s/kubernetesagentvars.t

```diff
@@ -9,7 +9,7 @@
 {{end}}
 {{if .IsWindows}}
     "winResourceNamePrefix" : "[substring(parameters('nameSuffix'), 0, 5)]",
```

Comment:
Let's reduce this to 4 to accommodate more than a single digit for node pools, i.e., to allow > 10 node pools.

Practically we don't know that > 10 node pools is tractable but it would be stupid to be blocked by the vm name algorithm when we do.